app/serializers/dependents_serializer.rb

```diff
@@ -1,6 +1,74 @@
 # frozen_string_literal: true
 
+START_EVENTS = %w[EMC SCHATTB].freeze
+LATER_START_EVENTS = %w[SCHATTB].freeze
+END_EVENTS = %w[T18 SCHATTT].freeze
+FUTURE_EVENTS = (LATER_START_EVENTS + END_EVENTS).freeze
+
+module DependentsHelper
+  def max_time(a, b)
+    a[:award_effective_date] <=> b[:award_effective_date]
+  end
+
+  def in_future(decision)
+    Time.zone.parse(decision[:award_effective_date]) > Time.zone.now
```

Comment:
Do you anticipate `award_effective_date` always being a valid date string? I'm just wondering if you need any error handling here.

Reply:
Yes, it is pulled from BGS directly and as far as I am aware this is always filled